src/elisp/treemacs-faces.el

```diff
@@ -35,6 +35,16 @@ Using this face is incompatible with `treemacs-git-mode' (exept for the simple
 variant), so it will only be used if git-mode is disabled or set to simple."
   :group 'treemacs-faces)
 
+(defface treemacs-all-the-icons-root-face
+  '((t (:inherit font-lock-string-face)))
+  "Face used for the root icon in all-the-icons theme."
+  :group 'treemacs-faces)
+
+(defface treemacs-all-the-icons-file-face
+  '((t (:inherit font-lock-doc-face :slant normal)))
```

Comment:
The slant looks like an adjustment for your own theme, so I don't think it should be a general setting.

Besides, shouldn't this inherit `treemacs-file-face` instead?

Reply:
Removed the `slant`.
I could not make it work inheriting from `treemacs-file-face`, maybe all-the-icons face needs foreground and background declared on the face?

Reply:
Strange, it works on my end.

Using doc-face here is for the best anyway, treemacs-file-face just inherits default, and icons should have some color.